esp/services/ws_store/espstorelib/daliKVStore.cpp

```diff
@@ -35,7 +35,7 @@ bool CDALIKVStore::createStore(const char * apptype, const char * storename, con
     VStringBuffer xpath("Store[%s='%s'][1]", DALI_KVSTORE_NAME_ATT,  storename);
     if (root->hasProp(xpath.str()))
     {
-        IWARNLOG("DALI KV Store: Cannot create app '%s' entry, it already exists", storename);
+        LOG(MCuserInfo,"DALI Keystore createStore(): '%s' entry already exists", storename);
```

Comment:
Is it going to be common to reach here / is it normal for it to already exist?
If so, I'd have thought better not to log each time, only when it is actually created, rather than every time already exists.
But if this code path is only expected to be hit on startup or similar (low-frequency) it makes sense as it.

Reply:
We were reaching this code because the base class blindly attempted to create pre-configured stores on startup. That logic has been updated. This function is exposed as a WebService method and can be accessed by end users, but it should be very infrequent. If it ends up being called many times, this log entry will help expose a problem in its usage.